apps/core/components/ItineraryCard/ItineraryCard.js

```diff
@@ -70,28 +63,23 @@ class ItineraryCard extends React.Component<Props, State> {
       <Card
         style={[
           styles.container,
-          (hovered || detailOpened) && styles.containerElevated,
+          (isHovered || detailOpened) && styles.containerElevated,
         ]}
       >
-        <Hoverable
-          onMouseEnter={this.handleOnMouseEnter}
-          onMouseLeave={this.handleOnMouseLeave}
-        >
-          <TouchableWithoutFeedback onPress={this.handlePress}>
-            <View>
-              <ItineraryCardWrapper
-                localizedPrice={localizedPrice}
-                detailOpened={detailOpened}
-              >
-                <ItineraryTypeRenderer
-                  typename={typename}
-                  oneWayComponent={<TripSectorOneWay itinerary={data} />}
-                  returnComponent={<TripSectorReturn itinerary={data} />}
-                />
-              </ItineraryCardWrapper>
-            </View>
-          </TouchableWithoutFeedback>
-        </Hoverable>
+        <TouchableWithoutFeedback onPress={this.handlePress} {...rest}>
+          <View>
```

Comment:
Probably not from your PR, but I will comment it anyway 😄. This could be replaced by a React.Fragment(`<>..</>`)

Reply:
Let's do it then

Reply:
Aha, ok, thanks for trying 🙂